### PR TITLE
Small changes on how-to-publish.md

### PR DESCRIPTION
**Pull request by eatdostacos**
### Description

Some more small corrections

### Fixes #

### CODE REVIEW

File: content/faq/how-to-publish.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Yes, the claim can be edited to increase the bid amount or you can also send a S
 
 ## My video doesn't stream in the app or on lbry.tv, what's wrong?
 
-The in-app and lbry.tv video player's streaming capability works best with the H264/AAC format, typically in a MP4 container but others like M4V will also work. Other non-streaming video types like AVI/WMV/MOV file formats are not supported by the in-app player, but they can be shared/downloaded and will play outside of the app.
+The in-app and lbry.tv video player's streaming capability works best with the H264/AAC format, typically in an MP4 container but others like M4V will also work. Other non-streaming video types like AVI/WMV/MOV file formats are not supported by the in-app player, but they can be shared/downloaded and will play outside of the app.
```
Comment (littlemadhatter) [inline content/faq/how-to-publish.md:122]: The only time you use "an" is if the after part is a vowel, A, E, I, O, U

Comment (eatdostacos) [reply content/faq/how-to-publish.md:122]: This is not true. On acronyms you have to see what the first letter is. If it's M you pronounce it "emm" and put an in front of it.

Comment (littlemadhatter) [reply content/faq/how-to-publish.md:122]: Didn't know this, thanks, sorry.

**Comment by tzarebczan** [comment]
Thanks!

**Pull request merged**